sdk/include/vcruntime/emmintrin.h

```diff
@@ -27,7 +27,9 @@ typedef union _DECLSPEC_INTRIN_TYPE _CRT_ALIGN(16) __m128i
     unsigned __int32 m128i_u32[4];
     unsigned __int64 m128i_u64[2];
 } __m128i;
+#ifdef _STATIC_ASSERT
```

Comment:
If STATIC_ASSERT isn't defined, this means this vcruntime header isn't included together with either [`vcruntime/crtdefs.h`](https://github.com/reactos/reactos/blob/d06d9f3e8f63e5a4b610e9605b40da20990e23a2/sdk/include/vcruntime/crtdefs.h#L144) or [`ucrt/corecrt.h`](https://github.com/reactos/reactos/blob/d06d9f3e8f63e5a4b610e9605b40da20990e23a2/sdk/include/ucrt/corecrt.h#L298) : is this really expected?

Reply:
yes, this is what happens when you build with wine's msvcrt headers.